src/actions/google/gcs/test_google_cloud_storage.ts

```diff
@@ -96,6 +96,35 @@ describe(`${action.constructor.name} unit tests`, () => {
           "A streaming action was sent incompatible data. The action must have a download url or an attachment.")
     })
 
+    it("errors if there is an upload error", () => {
+      const request = new Hub.ActionRequest()
+      request.type = Hub.ActionType.Dashboard
+      request.params = {
+        client_email: "myemail",
+        private_key: "mykey",
+        project_id: "myproject",
+      }
+      request.formParams = {
+        bucket: "mybucket",
+      }
+      request.attachment = {dataBuffer: Buffer.from("1,2,3,4", "utf8")}
+      return chai.expect(action.validateAndExecute(request)).to.eventually
+        .deep.equal({
+          refreshQuery: false,
+          success: false,
+          error: {
+            documentation_url: "TODO",
+            http_code: 500,
+            location: "ActionContainer",
+            message: "Internal server error. [Google Cloud Storage] Error while sending data Could not authenticate request\nerror:1E08010C:DECODER routines::unsupported",
```

Comment:
Wait is this making an actual call out to GCS?  We should probably update this test and make it local only or else it can become flaky and depend upon the implementation of the external service.